Makefile

```diff
@@ -24,6 +24,8 @@ ISO_VERSION ?= v$(VERSION_MAJOR).$(VERSION_MINOR).1
 # Dashes are valid in semver, but not Linux packaging. Use ~ to delimit alpha/beta
 DEB_VERSION ?= $(subst -,~,$(RAW_VERSION))
 RPM_VERSION ?= $(DEB_VERSION)
+# used by hyperkit versionCheck(whether user's hyperkit is older than this)
+HYPERKIT_VERSION ?= 0.20190201
```

Comment:
I am not sure if this is the best place to store the mininum required hyperkit version

Reply:
I think a better place for this is in the package itself as a constant

Reply:
I moved this constant.
Makefile(& pkg/version/version.go) → pkg/minikube/registry/drvs/hyperkit.go